.github/workflows/qa.yaml

```diff
@@ -52,8 +54,9 @@ jobs:
         uses: actions/setup-go@v2
         with:
           go-version: ${{ env.goversion }}
-      # We add a new user and execute under it for file permissions to be taken into account
-      # as root can write in RO directories.
+      # Workaround for https://github.com/actions/setup-go/issues/107
+      - name: Replace default go symlink
+        run: cp -f `which go` /usr/bin/go
```

Comment:
waow, this is a big issue IMHO. I think copying works because GOROOT is specified in some way by the action, but I really wonder if we will need to put that everywhere (why isn’t it failing in go vet and other checks?)